### PR TITLE
Integrate docker-compose

### PR DESCRIPTION
**Pull request by theCrius**

### CODE REVIEW

**Comment by Ponjimon** [comment]
Nice, thank you. Although there still is the issue with conflicting permissions due to the bind mount volume. It crashes for me starting it with docker-compose. Any idea how we could fix this?

**Comment by theCrius** [comment]
Can you paste the exact error log? It simply start on a fresh ubuntu 20.04 vm with only docker and docker-compose installed.

be sure to follow this guide: 
[Install Docker](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-20-04)
[Install Docker-Compose](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-20-04)

**Comment by Ponjimon** [comment]
The error is this:

```
UnauthorizedAccessException: Access to the path "Z:\vrising\server-data\Saves" is denied.
  at System.IO.Directory.CreateDirectoriesInternal (System.String path) [0x00000] in <00000000000000000000000000000000>:0
``` 

I think it's because of a mismatch between host and container permissions due to the Docker image assuming a GID and UID of 1000

**Comment by theCrius** [comment]
mhm, that is weird is definitely having something to do with the volume, you are right. Can you tell me what are the specs of your host? specifically which linux distro and what username are you connecting to with?

**Comment by Ponjimon** [comment]
It's running inside the devcontainer (see `.devcontainer` folder in this repo).

**Comment by theCrius** [comment]
As far as I know, the .devcontainer is something used to quickly spin a docker container from vscode so that you can manage and develop in an isolated environment. If it causes any issue, it's related to that and not the docker-compose itself.

Have you run this from a vscode instance running? On linux or windows (and if windows, WSL or host OS)?

**Comment by Ponjimon** [comment]
Oh, don't get me wrong, it has nothing to do with the compose file. I'm gonna merge your PR soon anyway :)

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services: 
+  vrising: 
+    image: public.ecr.aws/r0w6f3t3/vrising:latest
+    args:
+        STEAM_EPOC:
```
Comment (Ponjimon) [inline docker-compose.yaml]: I think you can remove this again. When I initially created the Dockerfile, I used another Dockerfile as the base and this was just a remnant.

**Comment by Ponjimon** [comment]
I updated the Docker image in `main` along with the readme. Once you update your branch, I will merge it

**Comment by theCrius** [comment]
Should be all good now, double check the readme just to be sure I haven't deleted the wrong lines while resolving the conflicts and we are good to go :)

**Comment by Ponjimon** [comment]
You removed your changes to the readme again :)

**Comment by theCrius** [comment]
FFfffff.... i'm having one of those days, bear with me

**Comment by theCrius** [comment]
All done now :)

File: README.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ Inside that volume, the server will store the save data and settings.
  ```console
  docker run -it -p 27015:27015/udp -p 27016:27016/udp -v /home/vrising/server-data:/vrising/server-data public.ecr.aws/r0w6f3t3/vrising:latest
  ```
+ 
+ Alternatively, you can use `docker-compose`. Simply run `docker-compose up -d`to spin up a container. The `docker-compose.yaml` file assumes that there is a directory `${home}/vrising-server` in which the container will mount a volum and store the saves and you can add your custom settings.
```
Comment (Ponjimon) [inline README.md]:
```suggestion
 Alternatively, you can use `docker-compose`. Simply run `docker-compose up -d` to spin up a container. The `docker-compose.yaml` file assumes that there is a directory `${HOME}/vrising-server` in which the container will mount a volume and store the saves where you can add your custom settings.
```

**Review by Ponjimon — CHANGES_REQUESTED** [review]
Just some typos and then we're good to go

**Review by Ponjimon — APPROVED** [review]

**Pull request merged**